### ISSUE TITLE
AIP tag upload error

### ISSUE THREAD

**Issue by kdahlo** [broadinstitute/seqr#4177]
When attempting to upload AIP tag set, get an error message for a subset of variants (see error output below). However, these variants are visible in seqr (see example screenshot below).

![AIP_upload_bug](https://github.com/broadinstitute/seqr/assets/6993450/e2b94bfd-ca46-47ef-9bba-34701b067153)
<img width="1407" alt="example_variant" src="https://github.com/broadinstitute/seqr/assets/6993450/41f895f3-a79f-443c-ae00-5a57e5ec3599">

**Comment by hanars** [comment]
RGP_1630 currently has search disabled, as do several GRU families which were recently moved form the HMB project. Because search is disabled, they still have records of variants that were previously saved but can not perform any additional searches, which means that they will be unable to load AIP results. 

We should add an improvement to the code to not require the ability to search for variants that have already been saved, which I will do. However, once search is reenabled for these families the loading will also fix itself, and that may happen before I get the code fix for this in, so I will keep you posted to let you know which happens first

Also in the future please follow the bug report template fully, including both screenshots and links to pages where you are experiencing issues.

**Comment by hanars** [comment]
Actually, looking into this I think it should not need to be fetching those variants if they are already tagged. Can you please provide the actual file you are loading so I can investigate further?

**Comment by hanars** [comment]
@kdahlo please provide the requested file in order to unblock this issue

**Comment by kdahlo** [comment]
[2024-03_seqr_summary.json](https://github.com/user-attachments/files/15962093/2024-03_seqr_summary.json)

**Comment by hanars** [comment]
Note: this is due to a bug in seqr where we are incorrectly mapping families to the ONT project, which is not the desired behavior

**Comment by hanars** [comment]
I have fixed seqr so it now requires a specific list of projects in the metadata in order to make sure seqr assigns the AIP tags to the correct families/projects. In this case, I added the following to the metadata section:
```
"projects": ["Rare Genomes Project_Genomes_GRU", "Rare Genomes Project_Genomes_HMB"]
```

There are now much fewer failures, including the RG_1630 failure is resolved. There are still 5 families that are failing to upload, but I confirmed all of them currently have search disabled as part of the HMB->GRU move, and that none of the variants are already tagged in those families. Once search is fuly reloaded for the GRU project, this should work
